src/containers/Plot/index.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import PlotlyPlot from "react-plotly.js";
+// import { ActionCreator } from "redux";
+
+import { State } from "../../state/types";
+import { PlotConfig } from "../ResultsPanel/types";
+import { getCurrentTime } from "../../state/selection/selectors";
+// import { changeTime } from "../../state/selection/actions";
+// import { ChangeTimeAction } from "../../state/selection/types";
+import { PLOT_STYLE } from "../ResultsPanel/constants";
+
+interface PlotProps {
+    plotConfig: PlotConfig;
+    time: number;
+    // changeTime: ActionCreator<ChangeTimeAction>;
+}
+
+const styles = require("./style.css");
+
+class Plot extends React.Component<PlotProps, {}> {
+    shouldComponentUpdate() {
+        return this.props.plotConfig.hasTimeIndicator;
+    }
+
+    public render(): JSX.Element | null {
+        const { plotConfig, time } = this.props;
+        const { hasTimeIndicator, data, layout } = plotConfig;
+        const TIME_INDICATOR_LINE = "timeIndicatorLine";
+
+        if (hasTimeIndicator && time !== 0) {
```

Comment:
I would call this prop `renderTimeIndicator` or `shouldRenderTimeIndicator` because it doesn't mean it has one, it means it should have one